packages/cfpb-atomic-component/src/utilities/transition/MaxHeightTransition.js

```diff
@@ -0,0 +1,129 @@
+// Required modules.
+import BaseTransition from '@cfpb/cfpb-atomic-component/src/utilities/transition/BaseTransition.js';
+import EventObserver from '@cfpb/cfpb-atomic-component/src/mixins/EventObserver.js';
+
+// Exported constants.
+const CLASSES = {
+  CSS_PROPERTY: 'max-height',
+  BASE_CLASS:   'u-max-height-transition',
+  MH_DEFAULT:   'u-max-height-default',
+  MH_SUMMARY:   'u-max-height-summary',
+  MH_ZERO:      'u-max-height-zero'
+};
+
+/**
+ * MoveTransition
+ * @class
+ *
+ * @classdesc Initializes new MoveTransition behavior.
+ *
+ * @param {HTMLNode} element
+ *   DOM element to apply move transition to.
+ * @returns {MaxHeightTransition} An instance.
+ */
+function MaxHeightTransition( element ) {
+  const _baseTransition = new BaseTransition( element, CLASSES );
+  let previousHeight;
+
+  /**
+   * @returns {MaxHeightTransition} An instance.
+   */
+  function init() {
+    _baseTransition.init();
+
+    element.style.maxHeight = element.scrollHeight + 'px';
+
+    const _transitionCompleteBinded = _transitionComplete.bind( this );
+    _baseTransition.addEventListener(
+      BaseTransition.END_EVENT,
+      _transitionCompleteBinded
+    );
+
+    return this;
+  }
+
+  /**
+   * Handle the end of a transition.
+   */
+  function _transitionComplete() {
+    this.dispatchEvent( BaseTransition.END_EVENT, { target: this } );
+
+    if ( element.scrollHeight > previousHeight ) {
+      element.style.maxHeight = element.scrollHeight + 'px';
+    }
+  }
+
+  /**
+   * Move to the element's original coordinates.
```

Comment:
There are a couple copy/paste leftovers to update

Reply:
Ahh oops! Should be fixed.